tasks/winbuildscripts/Generate-Chocolatey-Package.ps1

```diff
@@ -69,7 +61,7 @@ if (![string]::IsNullOrEmpty($VersionOverride)) {
 }
 $copyright = "Datadog {0}" -f (Get-Date).Year
 
-$releasePattern = "(\d+\.\d+\.\d+)"
+$releasePattern = "(\d+\.\d+\.\d+)$"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Should this start also with `^` ?